specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -125,7 +125,15 @@ on the VDK menu.
 You can find a video presenting a few UI components in the main directory of the VEP.
 
 ## API design
-No direct changes to the public API.
+
+A new API to access Job Input interfaces, `VDK.get_initialized_job_input()`, is available only when running a notebook. It is provided by `vdk-ipython` plugin.
```

Comment:
Should finalise_job() be added, as well?

Reply:
It is not new interface . Finalise job exists before

Reply:
Oh right yes

Reply:
done